src/components/Home/Home.js

```diff
@@ -23,8 +23,8 @@ class Home extends React.Component {
     this.props.history.replace('/disclaimer');
   };
 
-  error = () => {
-    this.props.history.replace('/error');
+  error = error => {
```

Comment:
Nice. Could we use this as an opportunity to rename `error` to `handleError`? Feels a bit easier to follow, else it could do anything: generate error message, throw an error, etc.